tests/testRoutesChangePassword.php

```diff
@@ -0,0 +1,209 @@
+<?php
```

Comment:
Copied from `tests/testRoutesGetUser.php`, generally the same except for success test